src/Tests/dotnet-sln.Tests/GivenDotnetSlnRemove.cs

```diff
@@ -735,5 +736,26 @@ public void WhenProjectIsRemovedThenDependenciesOnProjectAreAlsoRemoved()
             File.ReadAllText(solutionPath)
                 .Should().BeVisuallyEquivalentTo(ExpectedSlnContentsAfterRemoveProjectWithDependencies);
         }
+
+        [Fact]
+        public void WhenSolutionIsPassedAsProjectItPrintsSuggestionAndUsage()
+        {
+            var projectDirectory = _testAssetsManager
+                .CopyTestAsset("TestAppWithSlnAndCsprojFiles", identifier: "WhenSolutionIsPassedAsProject")
```

Comment:
I don't think the identifier is needed here (or any of the other tests)

Reply:
Ok will do, wasn't sure. I couldn't find any documentation on how this is supposed to work so have just been guessing.

Reply:
The identifier is a parameter used for uniquely identifying [Theory] tests vs [Fact] tests because they get put into a flat folder with the test name into a machine that runs them in parallel. Thanks for resolving it. I added documentation for this stuff a few months back as I had the same confusion!